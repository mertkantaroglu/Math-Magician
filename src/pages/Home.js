import '../App.css';

const Home = () => (
  <div>
    <section className="home">
      <h2>Welcome to our page!</h2>
      <p>
        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Eius ratione
        ex tempore error veniam voluptas consectetur ullam iste. Voluptatem
        accusamus harum numquam atque consectetur saepe architecto libero cum,
        totam dicta? Iste aspernatur rem quia. Voluptate iure magni minus
        animi assumenda. Ut laborum placeat nobis consequuntur qui ea ab
        assumenda voluptatem!Lorem ipsum dolor sit amet consectetur
        adipisicing elit. Quae qui ipsum magni impedit enim. Officiis iste
        saepe, ipsam minus veritatis architecto?
      </p>
      <p>
        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Eius ratione
        ex tempore error veniam voluptas consectetur ullam iste. Voluptatem
        accusamus harum numquam atque consectetur saepe architecto libero cum,
        totam dicta? Iste aspernatur rem quia. Voluptate iure magni minus
        animi assumenda. Ut laborum placeat nobis consequuntur qui ea ab
        assumenda voluptatem!Lorem ipsum dolor sit amet consectetur
        adipisicing elit.
      </p>
    </section>
  </div>
);

export default Home;
