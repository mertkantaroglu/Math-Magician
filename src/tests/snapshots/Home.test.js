import renderer from 'react-test-renderer';
import Home from '../../pages/Home';

describe('Home', () => {
  test('should match with Home snapshot', () => {
    const tree = renderer.create(
      <Home />,
    );
    expect(tree).toMatchSnapshot();
  });
});
