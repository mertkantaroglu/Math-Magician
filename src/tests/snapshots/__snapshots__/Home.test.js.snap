// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should match with Home snapshot 1`] = `
<div>
  <section
    className="home"
  >
    <h2>
      Welcome to our page!
    </h2>
    <p>
      Lorem ipsum dolor, sit amet consectetur adipisicing elit. Eius ratione ex tempore error veniam voluptas consectetur ullam iste. Voluptatem accusamus harum numquam atque consectetur saepe architecto libero cum, totam dicta? Iste aspernatur rem quia. Voluptate iure magni minus animi assumenda. Ut laborum placeat nobis consequuntur qui ea ab assumenda voluptatem!Lorem ipsum dolor sit amet consectetur adipisicing elit. Quae qui ipsum magni impedit enim. Officiis iste saepe, ipsam minus veritatis architecto?
    </p>
    <p>
      Lorem ipsum dolor, sit amet consectetur adipisicing elit. Eius ratione ex tempore error veniam voluptas consectetur ullam iste. Voluptatem accusamus harum numquam atque consectetur saepe architecto libero cum, totam dicta? Iste aspernatur rem quia. Voluptate iure magni minus animi assumenda. Ut laborum placeat nobis consequuntur qui ea ab assumenda voluptatem!Lorem ipsum dolor sit amet consectetur adipisicing elit.
    </p>
  </section>
</div>
`;
